apps/condo/domains/banking/schema/BankCategory.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Generated by `createschema banking.BankCategory 'name:Text;'`
+ */
+
+const { LocalizedText } = require('@open-condo/keystone/fields')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const access = require('@condo/domains/banking/access/BankCategory')
+
+
+const BankCategory = new GQLListSchema('BankCategory', {
+    schemaDoc: 'Expenses category, that will be associated with transactions and contractors',
```

Comment:
This model looks little bit strange. It only consists of `LocalizedText` field. May be it should also contain type category, that might be a constant?

Reply:
We have discussed the architectural part of the task.
It will act like a business-model.
It is simple yet, but in the future it may have hierarchy.
Hierarchy will be implemented as Nested Set, which is moved to separate PR.